tests/bootstrap.php

```diff
@@ -4,11 +4,11 @@
 	die( 'Not an entry point' );
 }
 
-error_reporting( E_ALL | E_STRICT );
-ini_set( 'display_errors', 1 );
+error_reporting( -1 );
+ini_set( 'display_errors', '1' );
 
 if ( !is_readable( __DIR__ . '/../vendor/autoload.php' ) ) {
 	die( 'You need to install this package with Composer before you can run the tests' );
 }
 
-require_once __DIR__ . '/../vendor/autoload.php';
+require __DIR__ . '/../vendor/autoload.php';
```

Comment:
What was wrong with `require_once`?

Reply:
Fails when bootstrap included twice

Reply:
How? Why is it important to be able to include the autoload.php file twice? Could it be that you confused the two "require_once" vs. "require" here?

Reply:
I do not recall the details. Me or someone else ran into a valid scenario where the bootstrap got included more than once, and an error occurred here. Hence been using just require in the last 30 or so PHP projects I poked at. My changes here are a copy paste from that version, minus the declare_strict, cause ancient PHP.